test/runtests.jl

```diff
@@ -26,11 +26,14 @@ core_tests = [
     "wigner.jl",
 ]
 
+# Include helper functions to avoid repetition
+include("helpers.jl")
+
 if (GROUP == "All") || (GROUP == "Code-Quality")
     using QuantumToolbox
     using Aqua, JET
 
-    include(joinpath(testdir, "core-test", "code_quality.jl"))
+    include(joinpath(testdir, "code-quality", "code_quality.jl"))
```

Comment:
This means we also need to add

```
- test/code-quality/**
```
to Line `12` and `22` in the file [`.github/workflows/Code-Quality.yml`](https://github.com/qutip/QuantumToolbox.jl/blob/main/.github/workflows/Code-Quality.yml)